go/apps/api/openapi/openapi.yaml

```
@@ -219,6 +218,39 @@ components:
             Omitting this field preserves the current enabled status.
             Useful for temporarily suspending access during billing issues, security incidents, or maintenance windows without losing key configuration.
           example: true
+        roles:
+          type: array
+          maxItems: 100 # Reasonable limit for role assignments per key
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)